src/core_plugins/kibana/public/management/react/lib/table_props.js

```diff
@@ -0,0 +1,92 @@
+import { Component } from 'react';
+import { sortBy as sortByLodash, chunk } from 'lodash';
+
+export class TableProps extends Component {
+  constructor(props) {
+    super(props);
+
+    const {
+      page = 0,
+      perPage = 10,
+      filterBy = {},
+      sortBy,
+      sortAsc = true
+    } = this.props;
+
+    this.state = {
```

Comment:
Copying props into local state is problematic, as changes to props aren't reflected in a component that is already mounted unless you wire it up with a `componentWillRecieveProps` lifecycle. If can be done, but generally speaking, this is a smell.